### PR TITLE
Additions to rpc getblockstats command

### PR DESCRIPTION
**Pull request by tomasbrod**
* now contains average block size (in kBytes)
`totals.blocksizek`: sum of serialized size of all analysed blocks
`averages.blocksizek`: average block size

### CODE REVIEW

File: src/rpcblockchain.cpp [hunk 1]
```diff
@@ -5419,6 +5426,8 @@ json_spirit::Value rpc_getblockstats(const json_spirit::Array& params, bool fHel
         result.push_back(Pair("first_time", TimestampToHRDate(l_first_time)));
         result.push_back(Pair("last_time", TimestampToHRDate(l_last_time)));
         result.push_back(Pair("time_span_hour", ((double)l_last_time-(double)l_first_time)/(double)3600));
+        result.push_back(Pair("min_blocksizek", size_min_blk/(double)1024));
```
Comment (denravonska) [inline src/rpcblockchain.cpp:5429]: You can use 1024.0 and skip the cast. Trying to keep it as cast clean as possible :)

Comment (tomasbrod) [reply src/rpcblockchain.cpp:5429]: Yeah.
Cast is used for dividing by block count in other places.
used cast to keep consistency

Comment (denravonska) [reply src/rpcblockchain.cpp:5429]: Please change to just a double literal since it needs to be done later anyway.

Comment (denravonska) [inline src/rpcblockchain.cpp]: Oooh now I get the k. Ignore that.

Comment (denravonska) [reply src/rpcblockchain.cpp:5429]: nvm, skip the double change. We'll fix the entire function later.

**Pull request merged**